src/App.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-env browser */
+import Layout from "@organisms/Layout"
+import React, { Suspense } from "react"
+import { useTranslation } from "react-i18next"
+import { Route, Routes } from "react-router-dom"
+import Home from "./pages"
+import NotFound from "./pages/NotFound"
+import About from "./pages/about"
+import Events from "./pages/events"
+import Projects from "./pages/projects"
+
+const App = () => {
+  const { i18n } = useTranslation()
+  const baseUrl = i18n.language === "it" ? "" : "/" + i18n.language
```

Comment:
Here an additional `/` is added, so when the language is not Italian the URL would end up being something like `example.org//en/about`. While this shouldn't create a problems per se, it's better to remove it

Reply:
To avoid weighing down the code and especially maybe to have problems in the future, I decided to leave the universal url, operating only a change of language in the content, so that we are sure that everything will be successful. Later we can try again maybe to do something like this, however I don’t think there is a better solution, using only `react-router-dom` and not `next`

Reply:
I think we should definitely look into [vite-plugin-ssr](https://vite-plugin-ssr.com/), as it seems a better [alternative to Next.js ](https://vite-plugin-ssr.com/nextjs-comparison) at least when using Vite, and also supports custom HTTP server (in our case Polito's ones) and not only the built in server that Next.js uses

Reply:
Okay perfect, later I’ll take a look, because I’ve never heard of it. Maybe for now we can close this PR and come back to translations later, if you agree. In this way we avoid to make now this merge and in future maybe another change always linked to the translation